res/build.sh

```diff
@@ -28,8 +28,28 @@ sed -e "s,@PREFIX@,$PREFIX," res/void-docs.in > res/void-docs
 sed -e "s,@PREFIX@,$PREFIX," res/void-docs.1.in > res/void-docs.1
 
 # Build PDF
+
 echo "Building PDF"
+
+## Temporarily add OpenType version of Latin Modern to font cache.
+
+TMP_FONTDIR=$(mktemp -d -p ~/.fonts)
+mkdir -p $TMP_FONTDIR
+cp /opt/texlive/2020/texmf-dist/fonts/opentype/public/lm/* $TMP_FONTDIR
```

Comment:
Isn't this from the `texlive-bin` package? `texlive-core` seems to have them in `/usr/share/texmf-dist/fonts/opentype/public/lm/`

Reply:
Ah, that's much better; i'll change this. Is (relatively) new due to the recent TeX Live packaging changes? i've been using TeX Live since first installing Void, and not sure whether i just missed this existing prior to the changes.